llvm/test/CodeGen/X86/tls-multi-use.ll

```diff
@@ -0,0 +1,149 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -o - %s | FileCheck %s
+; RUN: llc -o - -relocation-model=pic %s | FileCheck --check-prefix=CHECK_PIC %s
+target triple = "x86_64--linux-gnu"
+
+@foo = dso_local thread_local global i32 0, align 4
+@foo_nonlocal = thread_local global i32 0, align 4
```

Comment:
Not used?

Reply:
good catch, this was intended for the 2nd function so we see `dso_local` and non-`dso_local` variables referenced in the test.